common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1819,6 +1819,10 @@ public static enum ConfVars {
     HIVE_STRICT_CHECKS_BUCKETING("hive.strict.checks.bucketing", true,
         "Enabling strict bucketing checks disallows the following:\n" +
         "  Load into bucketed tables."),
+    HIVE_STRICT_CHECKS_OFFSET_NO_ORDERBY("hive.strict.checks.offset.no.orderby", false,
+        "Enabling strict offset checks disallows the following:\n" +
+        "  OFFSET without ORDER BY.\n" +
```

Comment:
2 white spaces. I think 1 would  be enough :)

Reply:
I followed other `HIVE_STRICT_*`. I guess it is a cosmetic purpose but I don't know.